code/modules/projectiles/boxes_magazines/internal_mag.dm

```diff
@@ -2,6 +2,7 @@
 
 /obj/item/ammo_box/magazine/internal
 	desc = "Oh god, this shouldn't be here"
+	flags_1 = CONDUCT_1|ABSTRACT_1
```

Comment:
Why even CONDUCT then?

Reply:
because it's in the parent type and i didn't want to mess with it since i don't know how its used

Reply:
CONDUCT means that the item conducts electricity. For an item that's an abstract never to be seen item, I don't think that matters.